<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>771</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_employee">
     <attribute name="title">
      <string>Client</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_ajouter_emp">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_id_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>30</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cin_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>CIN</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>130</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>NOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>PRENOM</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_tel_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>230</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>MATRICULE</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>30</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_cin_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>80</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_nom_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prenom_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>180</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_matricule_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>230</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter_c">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_quitter">
        <property name="geometry">
         <rect>
          <x>550</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_c">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>280</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir_c">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_email_c">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>280</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_adresse_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>280</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>EMAIL</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_afficher_emp">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>641</width>
          <height>301</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_c">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2_quitter">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_id">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_nom">
        <property name="geometry">
         <rect>
          <x>124</x>
          <y>10</y>
          <width>91</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par nom</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_prenom">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par prenom</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_cin">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>360</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher par matricule</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_email">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>360</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_chercher_adresse">
        <property name="geometry">
         <rect>
          <x>470</x>
          <y>360</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher par email</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_pdf">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Mail</string>
       </attribute>
       <widget class="QLineEdit" name="lineEdit_mail">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>70</y>
          <width>281</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="subject">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>130</y>
          <width>281</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QTextEdit" name="msg">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>210</y>
          <width>281</width>
          <height>85</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>70</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>À</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>130</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Objet</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>210</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_envoyer_mail">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>340</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Envoyer</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Parking</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>771</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_ajouter_emp_2">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QLabel" name="label_id_c_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>70</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>ID parking</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_cin_c_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>120</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date libre</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_nom_c_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Matricule</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prenom_c_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>220</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_tel_c_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>270</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_parking">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>70</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_date_libre">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_matricule_p">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>170</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>220</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_prix">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>270</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ajouter_parking">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_quitter_2">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_modifier_parking">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>340</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir_parking">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>30</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_afficher_emp_2">
       <attribute name="title">
        <string>Afficher</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>641</width>
          <height>311</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_supprimer_parking">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>320</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_rafraichir_2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Rafraichir</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2_quitter_2">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>360</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_id_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par ID</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_nom_2">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>10</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par type</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_tri_prenom_2">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>10</y>
          <width>81</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Tri par prix</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_matricule">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_cherche_matricule">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>360</y>
          <width>151</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher par matricule</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_chercher_type">
        <property name="geometry">
         <rect>
          <x>290</x>
          <y>360</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_chercher_type">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>360</y>
          <width>171</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Chercher par type</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_pdf_parking">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>671</width>
          <height>301</height>
         </rect>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>20</y>
          <width>191</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Afficher Statistique</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
